### PR TITLE
docs: add per manager known list of issues

### LINKED ISSUE #15546: Renovate documentation - Link relevant bugs and issues to package managers’ documentation
### What would you like Renovate to be able to do?

In the documentation of the Renovate supported package managers, I would like to add all related current open issues, both bugs and feature requests.

This will provide 2 main benefits:

- Better visibility for users regarding known bugs and limitations within the relevant package manager’s use cases, along with feature suggestions currently in the pipeline.

- Better documentation to be automatically published in each released version of the Renovate tool.

### If you have any ideas on how this should be implemented, please tell us here.

Go over GitHub’s issues list and filter them based on labels:

- Each supported package manager has its own designated label.

- Only issues that were scanned by a contributor and assigned a priority will be looked at (no issues of “priority-5-trige”).

image

Add an automatically updated segment in the documentation package manager web page that will list:

- Title of issue in GitHub (including issue number)

- Link to GitHub issue

- Indication wether the issue is a bug or a feature

image

### Is this a feature you are interested in implementing yourself?

No

### PR DESCRIPTION
**Pull request by Gabriel-Ladzaretti**
## Changes
Apply an API token to a previously merged PR

## Context
A follow up to -
 - #15936

Which reverts the following due to exceeded rate limit
 - #15791

Closes #15546

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: tools/docs/manager.ts [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 import type { RenovateConfig } from '../../lib/config/types';
+import { logger } from '../../lib/logger';
 import { getManagers } from '../../lib/modules/manager';
+import { GithubHttp } from '../../lib/util/http/github';
+import { getQueryString } from '../../lib/util/url';
 import { readFile, updateFile } from '../utils';
+import type { GithubApiQueryResponse, ItemsEntity } from './github-query-items';
 import { getDisplayName, getNameWithUrl, replaceContent } from './utils';
 
+const gitHubApiUrl = 'https://api.github.com/search/issues?';
```
Comment (viceice) [inline tools/docs/manager.ts:11]:
```suggestion
const gitHubApiUrl = 'https://api.github.com/search/issues?';

if (process.env.GITHUB_TOKEN) {
  hostRules.Add({ matchHost: 'api.github.com', token: process.env.GITHUB_TOKEN });
}
```

Something like this should do the trick. That way the `GithubHttp` should automatically use the token.

**Review by viceice — DISMISSED** [review]
🤞

File: tools/docs/manager.ts [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+
+if (process.env.GITHUB_TOKEN) {
+  hostRules.add({
```
Comment (viceice) [inline tools/docs/manager.ts:15]:
```suggestion
  logger.debug('Using GITHUB_TOKEN from env');
  hostRules.add({
```

just some vissibillity ?

**Comment by Gabriel-Ladzaretti** [comment]
looks like it passed on my end 
https://github.com/ladzaretti/renovatebot.github.io/pull/1

**Review by viceice — APPROVED** [review]

**Pull request merged**